articles/active-directory-b2c/faq.yml

```diff
@@ -210,7 +210,7 @@ sections:
           1. Under **Access management for Azure resources**, select **Yes**, and then select **Save**.
           1. Sign out of the Azure portal and then sign back in to refresh your access.
           1. Select **Azure Active Directory** on the left-hand menu.
-          1. On the **Overview** page, select **Delete directory**. Follow the on-screen instructions to complete the process.
+          1. On the **Overview** page, select  **Manage Tenants**. Check the tenant and select **Delete**. Follow the on-screen instructions to complete the process.
```

Comment:
```suggestion
          1. In the left menu, select **Overview**, and then select **Manage Tenants**. Checkmark the tenant you want to delete, and then select **Delete**. Follow the on-screen instructions to complete the process.
```

Reply:
See my suggested commit, if it makes sense, commit it, and I'll sign off the RP.

Reply:
@leobastin Please commit my suggested changes.